src/pages/level/Level.tsx

```diff
@@ -1,19 +1,51 @@
 import * as yup from "yup"
 import { Box, Typography } from "@mui/material"
-import { type FC, useState } from "react"
+import { type FC, useEffect, useState } from "react"
+import Cookies from "js-cookie"
 import { useParamsRequired } from "codeforlife/hooks"
 
 import Controls from "./Controls"
 import { paths } from "../../routes"
 
 export interface LevelProps {}
 
+const COOKIE_NAME = "Rapid Router"
+
+const TwoPanelsLayoutOptions = {
+  Auto: "2.Auto",
+  VerticalSplit: "2.VerticalSplit",
+  HorizontalSplit: "2.HorizontalSplit",
+} as const
+
+// Value temporarily unused
+// eslint-disable-next-line
+const ThreePanelsLayoutOptions = {
+  Auto: "3.Auto",
+  VerticalSplitWithLeftHorizontal: "3.VerticalSplitWithLeftHorizontal",
+  VerticalSplit: "3.VerticalSplit",
+  HorizontalSplit: "3.HorizontalSplit",
+} as const
+
+type LayoutOption =
+  | (typeof ThreePanelsLayoutOptions)[keyof typeof ThreePanelsLayoutOptions]
+  | (typeof TwoPanelsLayoutOptions)[keyof typeof TwoPanelsLayoutOptions]
+
 interface LevelState {
   panels: 2 | 3
+  layoutOption: LayoutOption
 }
 
 const Level: FC<LevelProps> = () => {
-  const [level] = useState<LevelState>({ panels: 2 })
+  const [level] = useState<LevelState>({
+    panels: 2,
+    layoutOption: TwoPanelsLayoutOptions.Auto,
+  })
+
+  useEffect(() => {
+    if (!Cookies.get(COOKIE_NAME)) {
+      Cookies.set(COOKIE_NAME, JSON.stringify(level))
+    }
+  }, [level])
```

Comment:
Your idea of using a hook was excellent, do that!
```suggestion
  const settings = useSettings()
```